.github/workflows/docsbuild-release.yaml

```
@@ -0,0 +1,73 @@
+name: ansible-release-porting-guide
+on:
+  workflow_dispatch:
+    inputs:
+      ansible_version:
```

The de-facto convention for variable names in GHA is using kebab-case, this allows distinguishing between variables of the workflow defitnion and others (inline like shell or python scripts)
```suggestion
      ansible-version:
```